src/aspire/source/simulation.py

```diff
@@ -477,46 +477,60 @@ def eval_coords(self, mean_vol, eig_vols, coords_est):
 
         :param mean_vol: A mean volume in the form of a Volume instance.
         :param eig_vols: A set of eigenvolumes in an Volume instance.
-        :param coords_est: The estimated coordinates in the affine space defined centered at `mean_vol` and spanned
-            by `eig_vols`.
-        :return:
+        :param coords_est: The estimated coordinates in the affine space defined centered
+            at `mean_vol` and spanned by `eig_vols`.
+        :return: Dictionary containing error, relative error, and correlation for each set
+            of estimated coordinates.
         """
         assert isinstance(mean_vol, Volume)
         assert isinstance(eig_vols, Volume)
         coords_true, res_norms, res_inners = self.vol_coords(mean_vol, eig_vols)
 
         # 0-indexed states vector
         states = self.states - 1
-        coords_true = coords_true[states]
+
+        coords_true = coords_true.T[states]
```

Comment:
This transpose, is it due to the recent convention changes or an old bug? Thanks

Reply:
Just a bug. For `n_vols = 2`, `coords_true` is 1D so the bug wasn't tickled until we tried more than 2 vols.

Reply:
Perfect, thank you.